src/Task/useCases/getTasks/GetTasksUseCase.ts

```diff
@@ -0,0 +1,17 @@
+import { GetTasksResponse } from './GetTasksResponse';
+import { GetTasksRepoInterface } from '../../repos/getTasks/IGetTasksRepository';
+
+export class GetTasksUseCase {
+  constructor(private readonly taskRepo: GetTasksRepoInterface) {}
+
+  async execute(): Promise<GetTasksResponse> {
+    try {
+      const resultTask = await this.taskRepo.getTasks();
+      return {
+        tasks: resultTask,
+      };
+    } catch (error) {
+      throw new Error('Something went wrong. Try again!');
```

Comment:
For my opion, I would just call the taskRepo function and let it throw exception, then we can catch the error in controller.
Moreover, we should throw + log the original error, or else we cannot know what the real error is

Reply:
I have fixed :) and will do proper logging in the next fixs.